src/index.html

```diff
@@ -4,14 +4,144 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalabel=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset ">
+        <legend>Personal information</legend>
+        <label  class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+          >
+        </label>
+        <label name="perinfo" class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="0"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="perinfo"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset ">
+        <legend>Registration</legend>
+        <label name="email" class="label">
+              E-mail:
+          <input
+             type="email"
+             name="login"
+             placeholder="email@example.com"
+             required
+             autocomplete="off"
+          >
```

Comment:
```suggestion
        <label name="email" class="label">
           E-mail:
          <input
             type="email"
             name="login"
             placeholder="email@example.com"
             required
             autocomplete="off"
          >
```

Reply:
FIx identations following rules described [here](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Reply:
Fix it everywhere